gems/pending/util/miq-xml.rb

```diff
@@ -5,43 +5,34 @@
 require 'util/xml/xml_diff'
 require 'util/xml/xml_patch'
 
-class MiqXml  
-	#MIQ_XML_VERSION = 1.0
-	#MIQ_XML_VERSION = 1.1	# Added create_time to root in seconds for easier time conversions
-	MIQ_XML_VERSION = 2.0	# Changed sub-xmls, added namespaces
+class MiqXml

```

Comment:
Oh, yes, definitely. But yes, I would like to relegate that to another PR. There are a bunch of files that could stand to be modified in gems/pending. :)